src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,47 @@
 
 class Animal {
   // write your code here
+
+  constructor(name, health = 100) {
+    this.health = health;
+    this.name = name;
```

Comment:
all amimals should have 100 health BY DEFAULT
```suggestion
  constructor(name) {
    this.health = 100;
    this.name = name;
```